tree/tree/src/TTree.cxx

```diff
@@ -6244,6 +6244,9 @@ Long64_t TTree::LoadTree(Long64_t entry)
       return -1;
    }
 
+   // create cache if wanted
+   if (fCacheDoAutoInit) SetCacheSizeAux();
```

Comment:
Could you get used to moving the "then" part into a separate line? That makes future breakpoint setting a lot easier.